rpcs3/Emu/NP/fb_helpers.cpp

```diff
@@ -277,6 +255,52 @@ u16 np_handler::RoomDataInternal_to_SceNpMatching2RoomDataInternal(const RoomDat
 	return member_id;
 }
 
+void np_handler::RoomMemberDataInternal_to_SceNpMatching2RoomMemberDataInternal(const RoomMemberDataInternal* member_data, const SceNpMatching2RoomDataInternal* room_info, SceNpMatching2RoomMemberDataInternal* sce_member_data)
+{
+	UserInfo2_to_SceNpUserInfo2(member_data->userInfo(), &sce_member_data->userInfo);
+	sce_member_data->joinDate.tick = member_data->joinDate();
+	sce_member_data->memberId      = member_data->memberId();
+	sce_member_data->teamId        = member_data->teamId();
+
+	// Look for id
+	if (member_data->roomGroup() != 0 && room_info)
+	{
+		bool found = false;
+		for (u32 g_index = 0; g_index < room_info->roomGroupNum; g_index++)
+		{
+			if (room_info->roomGroup[g_index].groupId == member_data->roomGroup())
+			{
+				sce_member_data->roomGroup = vm::cast(room_info->roomGroup.addr() + (u32{sizeof(SceNpMatching2RoomGroup)} * g_index));
+				found = true;
```

Comment:
missing a break, unless intended

Reply:
this is moved code. i'll add a break anyway since seems it should be fine